feconf.py

```diff
@@ -348,6 +349,14 @@ def get_empty_ratings():
     },
 }
 
+# Ids of the allowed classification algorithm classes.
```

Comment:
Can you comment a bit about how the dicts need to be structured in this list? It's not immediately obvious what name/algorithm_ids are. If you can refer to somewhere else in code that explains it, that's fine, too. Either way, it should be easier to understand what this list contains and how to change it.